lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -385,6 +400,82 @@ export class StandardController implements IController {
         });
     }
 
+    protected async getPlatformAuthProvider(
```

Comment:
nit: Return the platformAuthProvider or call this `setPlatformAuthProvider` instead